netcdf/dimensions.go

```diff
@@ -55,3 +55,7 @@ func (ds Dataset) Dim(name string) (d Dim, err error) {
 	d = Dim{ds, id}
 	return
 }
+
+func (dim Dim) Id() int {
```

Comment:
This should be called `ID`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Can you also add a test?

Reply:
I see. We can leave out the test for now.